.gitignore

```diff
@@ -1 +1,2 @@
 /target
+.env
```

Comment:
Why this change ? 

Just don't push your tokens to your fork

Reply:
Fixed in https://github.com/ayrat555/el_monitorro/pull/255/commits/1c44008e20d3220ef61e44e9a7393cb56758d831